service/migration_manager.cc

```diff
@@ -617,8 +617,18 @@ future<> migration_manager::announce_new_column_family(schema_ptr cfm, api::time
         }
 
         mlogger.info("Create new ColumnFamily: {}", cfm);
-        auto mutations = db::schema_tables::make_create_table_mutations(keyspace.metadata(), cfm, timestamp);
-        return include_keyspace_and_announce(*keyspace.metadata(), std::move(mutations), announce_locally);
+
+        auto ksm = keyspace.metadata();
+        return seastar::async([this, cfm, timestamp, ksm] {
```

Comment:
Because "on_before_create_column_family" is, like all listener functions, declared to be in a seastar::thread (for uniformity). 
It is not really needed in our use case, _yet_, but I prefer to make the change slightly future-proof (pun intended).